modules/schema/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "scripts": {
     "build-schema": "node ./build-schema",
-    "build-protos": "pbjs --keep-case --target json --proto-path ../.. --out dist/proto.v2.json ../../xviz/v2/*.proto",
+    "build-protos": "../../node_modules/.bin/pbjs --keep-case --target json --proto-path ../.. --out dist/proto.v2.json ../../xviz/v2/*.proto",
```

Comment:
@xiaoji should this be necessary?  It seemed to be referencing a global pbjs rather than the local one.  But I thought npm/yarn scripts should add the local node_modules/.bin to the PATH.  Now sure how this works with lerna.

Reply:
What happens if you use `yarn pbjs` here instead?